merfi/backends/rpm_sign.py

```diff
@@ -55,5 +53,9 @@ def sign(self):
                 logger.info('[CHECKMODE] signed: %s' % new_gpg_path)
                 logger.info('[CHECKMODE] signed: %s' % new_in_path)
             else:
-                # XXX do actual signing here
-                pass
+                os.chdir(os.path.dirname(path))
+                detached = ['rpm-sign', '--key', self.key, '--detachsign', 'Release', '--output', 'Release.gpg']
+                clearsign = ['rpm-sign', '--key', self.key, '--clearsign', 'Release', '--output', 'InRelease']
```

Comment:
as discussed in IRC, `--output` doesn't work with `--clearsign`. I was hoping it would, but it does not :cry: 

so we'll need to make this write its output to `InRelease` somehow